apps/src/weblab2/helpers/aiTutorHelper.ts

```diff
@@ -1,62 +1,15 @@
-import {FontAwesomeV6IconProps} from '@code-dot-org/component-library/fontAwesomeV6Icon';
-
-import experiments from '@cdo/apps/util/experiments';
-import weblab2I18n from '@cdo/apps/weblab2/locale';
-
 export const DEFAULT_AI_TUTOR_MODE = 'suggest';
 
-const MODE_MAP = {
-  suggest: {displayName: weblab2I18n.suggest(), iconName: 'comment-dots'},
-  outline: {displayName: weblab2I18n.outline(), iconName: 'diagram-project'},
-  guide: {displayName: weblab2I18n.guide(), iconName: 'compass'},
-  produce: {displayName: weblab2I18n.produce(), iconName: 'hammer'},
-};
+const MODES = ['suggest', 'outline', 'guide', 'produce'];
 
 // Given a mode, return the corresponding prompt name. If the mode is not
 // a possible mode, return the default prompt name.
 export const getPromptNameFromMode = (mode: string | undefined) => {
   const prefix = 'weblab2-';
   let suffix = 'suggest';
-  if (mode && Object.keys(MODE_MAP).includes(mode)) {
+  if (mode && MODES.includes(mode)) {
     suffix = mode;
   }
-  if (
-    experiments.isEnabledAllowingQueryString(
-      experiments.WEBLAB2_STRUCTURED_OUTPUT
-    ) ||
-    experiments.isEnabledAllowingQueryString(experiments.WEBLAB2_ACCEPT_REJECT)
-  ) {
-    suffix += '-structured';
-  }
+  suffix += '-structured';
   return `${prefix}${suffix}`;
 };
-
-// Given a list of modes, return the corresponding system prompt options.
-// Any mode in the list that is not a valid mode will be ignored.
-// If there are no valid modes in the list, return a single option with the
-// default mode.
-export const getPromptOptionsFromModes = (modes: string[]) => {
```

Comment:
This is no longer used now that we only use 1 mode per level (done in #68707), cleaning it up here.